pkg/cmd/dcos.go

```diff
@@ -175,6 +190,18 @@ func newPluginCommand(ctx api.Context, cmd plugin.Command) *cobra.Command {
 	}
 }
 
+// extractCorePlugin extracts the bundled core plugin into the plugins folder.
+func extractCorePlugin(ctx api.Context, cluster *config.Cluster) (*plugin.Plugin, error) {
+	ctx.Logger().Warn("Extracting “dcos-core-cli“...")
```

Comment:
The first quotes seem wrong.

Reply:
I used the one from here : https://github.com/dcos/dcos-cli/blob/master/pkg/setup/setup.go#L310

However we use normal quotes a bit below : https://github.com/dcos/dcos-cli/blob/master/pkg/setup/setup.go#L335

I don't have a strong opinion on using the one or the other, whatever we decide should be added to the style guide, it's not directly related to this PR though.

Reply:
Please use the one from https://github.com/dcos/dcos-cli/blob/master/pkg/setup/setup.go#L335 and fix https://github.com/dcos/dcos-cli/blob/master/pkg/setup/setup.go#L310 in this PR or another to follow US standards 
![quotation marks over the world](https://user-images.githubusercontent.com/2117580/46485635-5cf44b00-c7fc-11e8-8c4d-ddcee77dcdf9.png)

Reply:
I've updated it to use normal quotes on this line only, if you want to standardize quoting in our style guide please follow-up with a separate jira as it's not directly related to this PR.

Reply:
Sure, thanks!